### ISSUE TITLE
Text Not Displaying - Ridesims.com

### ISSUE THREAD

**Issue by JamesAllgood** [ruffle-rs/ruffle#1136]
Hi Guys, 

I know a lot of work has happened the past few weeks to do with text not being displayed. 

On one of my swf files, the text isn't be displayed. 

https://ridesims.com/ride-simulators/Untamed

<img width="547" alt="Screenshot 2020-09-07 at 15 23 16" src="https://user-images.githubusercontent.com/10291664/92399161-ba4c5180-f121-11ea-81ca-a4aa157b94c7.png">

**Comment by Dinnerbone** [comment]
Relevant code: DefineSprite 818, frame 2:
```
tellTarget("..")
{
   if(_root.Language == 1)
   {
      Message = "CLICK TO POWER RIDE.";
   }
   else
   {
      Message = "KLIK OM DE SPANNING IN TE SCHAKELEN.";
   }
   if(_root.Power == 0)
   {
      gotoAndStop(2);
   }
   else
   {
      gotoAndStop(1);
   }
}
gotoAndPlay(1);
```


There's a bunch of warnings in the log about `".."` not being a valid target, but I'm not sure if that's directly related (it's in every game on ridesims)

**Comment by JamesAllgood** [comment]
is there any news on these text issues? I've seen a lot of text tickets raised in the past few days, didn't know if they where all connected

**Comment by Dinnerbone** [comment]
Nothing yet, still trying to diagnose this sorry!

**Comment by Herschel** [comment]
Fixed by #1201.
